src/ui-test/camelProjectOfferingTest.ts

```diff
@@ -0,0 +1,78 @@
+import { QuickOpenBox, InputBox, QuickPickItem } from "vscode-extension-tester";
+
+import { openCommandPrompt, typeCommandConfirm, convertArrayObjectsToText, convertArrayObjectsToTextAndDescription } from "./common/commonUtils";
+
+import { assertEqualOptions } from "./common/testUtils";
+
+import { ProjectInitializer } from "./common/projectInitializerConstants";
+import { Catalog } from "../Catalog";
+import { expect } from "chai";
+
+const CAMEL_MISSIONS_EXPECTED = [
+    'circuit-breaker', 'configmap', 'health-check', 'rest-http', 'istio-distributed-tracing'
+];
+
+/**
+ * @author odockal@redhat.com
+ */
+export function testFuseProjectOffering() {
+    describe('Verify Project initializer Camel/Fuse Command palette options', async function() {
+
+        let inputBox: QuickOpenBox;
+        let catalogFuse: any;
+
+        before(async function() {
+            this.timeout(5000);
```

Comment:
Can you try to use a conditional wait?
Have you spotted something missing to use ones?